src/explorer/appServiceNode.ts

```diff
@@ -85,6 +85,34 @@ export class AppServiceNode extends NodeBase {
         return this.start();
     }
 
+    async delete(azureAccount: AzureAccountWrapper): Promise<boolean> {
+        let servicePlanName = this.site.serverFarmId.substring(this.site.serverFarmId.lastIndexOf('/serverfarms/') + '/serverfarms/'.length);
+        let servicePlanRG = this.site.serverFarmId.substring(this.site.serverFarmId.indexOf('resourceGroups/') + 'resourceGroups/'.length, this.site.serverFarmId.indexOf('/providers/'));
+        let servicePlan = await this.getWebSiteManagementClient(azureAccount).appServicePlans.get(servicePlanRG, servicePlanName);
+        let options = {};
+        options['deleteEmptyServerFarms'] = false;
+        let mOptions: MessageOptions = { modal: true };
+        if (servicePlan.numberOfSites < 2) {
+            let input = await window.showInformationMessage(`This is the last web app on the plan, "${servicePlanName}".  Delete the Web App and App Service Plan?`, mOptions, ...['Both', 'Web App ONLY']);
+            if (input) {
+                let deleteServicePlan = false;
+                if (input === 'Both') {
+                    deleteServicePlan = true;
+                }
+                await this.getWebSiteManagementClient(azureAccount).webApps.deleteMethod(this.site.resourceGroup, this.site.name, { deleteEmptyServerFarm: deleteServicePlan });
+                return true;
+            }
+            return false;
+        } else {
+            let input = await window.showInformationMessage(`Delete the Web App "${this.site.name}"?`, mOptions, ...['Confirm']);
```

Comment:
Instead of `...['Confirm']` you can just do `'Confirm'`

Reply:
On the portal the message is: Are you sure you want to delete "${this.site.name}"

Would be nice if we can align with the portal.